zest/releaser/pypi.py

```diff
@@ -407,6 +409,31 @@ def version_levels(self):
             return default
         return result
 
+    def tag_format(self):
+        """Return the string format for the tag name used in the release.
+
+        Override the default ``%(version)s`` in ~/.pypirc or setup.cfg using
+        a ``tag-format`` option::
+
+            [zest.releaser]
+            tag-format = v%(version)s
+
+        Returns default of ``%(version)s`` when nothing has been configured.
+        """
+        default = '%(version)s'
+        if self.config is None:
+            return default
+        try:
+            result = self.config.get(
+                'zest.releaser', 'tag-format')
+        except (NoSectionError, NoOptionError, ValueError):
+            return default
+        # test the formatter
+        if not re.search('%\(version\)s', result):
+            print('%(version)s needs to be part of the string formatter')
```

Comment:
One small remaining tweak: be more specific about what *exactly* is wrong ('tag-format') and print the current value (in case one config file overrides another and you're left wondering what's wrong :-) ):

  print("%%(version)s needs to be part of 'tag-format': %s" % result)

I hope I have escaped the first `%` in the right way.